zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -37,6 +38,70 @@
 import zaza.openstack.utilities.openstack as zaza_openstack
 
 
+def add_storage(unit, label, pool, size):
+    """Add storage to a Juju unit.
```

Comment:
These functions look like they might go better in `zaza/openstack/utilities/juju.py` as they seem to be generic and not specific to the ceph tests?